src/kata6.joinNames.js

```diff
@@ -1,3 +1,13 @@
-const joinNames = (namesObj) => {};
+const joinNames = (namesObj) => {
+const arrayOfNames = namesObj.map(person => person.name);
+if (arrayOfNames.length > 1) {
+const finalName = arrayOfNames[arrayOfNames.length-1];
+const removeFinalName = arrayOfNames.splice(arrayOfNames.length-1);
```

Comment:
You don't have to assign the method to a variable if you don't use the variable again. 
It could just be 
`````
arrayOfNames.splice(arrayOfNames.length-1);
``````
at the same position.